CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 # Changelog
 
 ## [next]
-
+-fix(borderOpacityWhenMoving): "borderOpacityWhenMoving" does not take effect on the child shapes within the group [#9374](https://github.com/fabricjs/fabric.js/issues/9734)
```

Comment:
fix(InteractiveObject): .... then description is ok

Reply:
done